### DISCUSSION TITLE
Vectorized epilogue in v3 API

### DISCUSSION THREAD

**Discussion by masahi** [NVIDIA/cutlass#819] [Q&A] [answered]
Hi, I've spent some time studying the v3 API. One of the things that's not clear to me is how vectorized epilogues would work in the v3 API. My understanding is that existing implementations in https://github.com/NVIDIA/cutlass/tree/master/include/cutlass/epilogue/collective only support scalar epilogue. 

Since vectorized epilogues operate on the `Array<T, N>` type, do we need to "convert" `cute::Tensor` to `Array<T, N>`? Or will vectorized epilogue be natively supported on `cute::Tensor` slices in a future release?

**Comment by hwu36** [comment, marked as answer]
Stay tuned.  It is coming

**Comment by thakkarV** [comment]
Hi! have you seen example 50 in the meantime?

**Reply by masahi** [reply]
Yes, I know it vectorizes copying but not math. And my target is sm80 for now.
